org.lflang/src/org/lflang/generator/c/CGenerator.java

```diff
@@ -2250,10 +2139,19 @@ protected void setUpGeneralParameters() {
         }
         if (hasModalReactors) {
             // So that each separate compile knows about modal reactors, do this:
-            targetConfig.compileDefinitions.put("MODAL_REACTORS", "");
+            targetConfig.compileDefinitions.put("MODAL_REACTORS", "TRUE");
         }
-        if (targetConfig.threading) {
+        if (targetConfig.threading) {  // FIXME: This logic is duplicated in CMake
```

Comment:
In light of my decreased availability, I would like to expedite the process of merging this, even if we have to turn this comment into an issue.

Reply:
Let's turn it into an issue.